ycmd/tests/subcommands_test.py

```diff
@@ -24,17 +24,20 @@
                           PathToTestFile,
```

Comment:
On a related note, we _really_ need to refactor this file so that it's smaller. Similar for the completions tests. Probably in per-language files.

Not in this PR though.